typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -373,12 +373,23 @@ export class HyperlaneIsmFactory extends HyperlaneApp<ProxyFactoryFactories> {
           destination,
           config.owner,
         );
-        const tx = await domainRoutingIsmFactory.deploy(
+        // estimate gas
+        const estimatedGas = await domainRoutingIsmFactory.estimateGas.deploy(
```

Comment:
do we need to worry about similar changes in this file in the cli v2 branch? or will this be effectively deprecated

asking bc it may be nice to have a more reusable function that does this estimation and 10% bumping, but if it's gonna be replaced by cli-2.0 changes anyways then it's not worth it

Reply:
I've made a PR already that will include this bump: https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/3957

The logic's already in the multiprovider, the ISM deploy functions here are an edge case really

having a reusable func is probs good though, but I'll save that for the cli-2.0 PR